atd-toolbox/data_model/python/10_make_core_tables.py

```diff
@@ -90,6 +90,8 @@ def make_column_sql(columns, full_table_name):
             elif full_table_name == "charges":
                 fk_table_name = "people"
             fk_cascade = "on update cascade on delete cascade"
+        elif constraint and "not null default false" in constraint and "_edits" in full_table_name:
+            constraint = ""
```

Comment:
fixes a bug in our DB schema in which we had default values in our `_edits` schema

Reply:
im a bit confused about what `constraint` means here

Reply:
sorry, the code for these migration helper scripts is pretty spaghettified. 

This script is parsing something like this:

```json
 {
    "column_name": "prsn_exp_homelessness",
    "data_type": "boolean",
    "data_type_constraint": "not null default false",
    "foreign_column_cascade": "",
    "foreign_column_name": "",
    "foreign_table_name": "",
    "foreign_table_schema": "",
    "is_cris_column": true,
    "is_edits_column": true,
    "is_existing_foreign_table": false,
    "is_primary_key": false,
    "is_unified_column": true,
    "note": "",
    "record_type": "people",
    "source": "cris"
  }
```

and turning it into column sql like 

```
prsn_exp_homelessness boolean not null default false
```

so `constraint` in this case is whatever text is in `data_type_constraint` in the json blob

Reply:
but in the `people_edits` table we don't want the column to have a default value, so i added that extract logic to remove the default value for boolean columns in the `_edits` tables.